packages/nc-gui-v2/plugins/directives.ts

```diff
@@ -0,0 +1,59 @@
+import { defineNuxtPlugin } from '#app'
```

Comment:
Use a more specific name for this plugin instead of `directives`.
Possibly `resizeDirective` or something, so others know what the files is about by looking at the name.